devel/libert_util/src/regression.c

```diff
@@ -97,13 +97,27 @@ double regression_unscale(const matrix_type * beta , const matrix_type * X_norm
    beta = inv(X'�X)�X'�y
 */
 
-void regression_OLS( const matrix_type * X , const matrix_type * Y , matrix_type * beta) {
+void regression_OLS( const matrix_type * X , const matrix_type * Y , const matrix_type* E, matrix_type * beta) {
```

Comment:
I am quite skeptical to these changes which I feel are quite special case treatments added to the general regression function; if possible I would *really* prefer if the necessary fencing was done in the `fwd_step_enkf.c` implementation.

I do not really understand the linear algebra here, to me it seems like:
```latex
inv( X' X ) -> inv( X'X + E'E)? 
```